src/model-interfaces.ts

```diff
@@ -295,11 +295,11 @@ export interface HuskelappModell {
 }
 
 export interface TiltakshendelseModell {
-    hendelseId: string;
-    hendelseOpprettet: Date;
-    hendelseTekst: string;
-    hendelseLenke: string;
-    tiltakstypeKode: string | null;
+    id: string;
+    opprettet: Date;
+    tekst: string;
+    lenke: string;
+    tiltakstype: string | null;
```

Comment:
Jeg syns ikke vi skal fjerne `kode` fra navnet såfremt det er koden vi sender og ikke hele tiltakstypenavnet. Tror det blir mer forståelig når vi skal legge tilbake type-kolonnen og må bruke koden for å få typen!